src/PyPlot.jl

```diff
@@ -134,7 +134,11 @@ function find_backend(matplotlib::PyObject)
                 if defaultgui == :qt_pyside
                     pyimport_conda("PySide", "pyside")
                 else
-                    pyimport_conda("PyQt4", "pyqt")
+                    try
+                        pyimport_conda("PyQt5", "pyqt")
+                    catch
+                        pyimport_conda("PyQt4", "pyqt")
```

Comment:
Probably the second line should just be `pyimport("PyQt4")`, since the first `pyimport_conda` line will already install `pyqt` if it fails.